src/Integrations/Integrations/Symfony/SymfonyIntegration.php

```diff
@@ -82,6 +82,77 @@ public function init()
             ]
         );
 
+        \DDTrace\hook_method(
+            'Doctrine\ORM\UnitOfWork',
+            'executeInserts',
+            function ($This, $scope, $args) {
+                  $metadataClass = 'Doctrine\ORM\Mapping\ClassMetadata';
+                  if (
+                       !$args ||
+                       !isset($args[0]) ||
+                       !$args[0] ||
+                       !($args[0] instanceof $metadataClass)
+                  ) {
+                       return;
+                  }
+
+                  $entities = $This->getScheduledEntityInsertions();
+                  $userInterface = 'Symfony\Component\Security\Core\User\UserInterface';
+                  $found = 0;
+                  $userEntity = null;
+                  foreach ($entities as $entity) {
+                    if (! ($entity instanceof $userInterface)) {
+                        continue;
+                    }
+                    $found++;
+                    $userEntity = $entity;
+                  }
+
+                  if ($found != 1) {
+                    return;
+                  }
+
+                  \datadog\appsec\track_user_signup_event($userEntity->getUsername(), [], true);
```

Comment:
function_exists check missing

Reply:
Good one! I have just changed it and also used this opportunity to change it on the Laravel integration we already had